playbooks/files/.github/commitlint.config.js

```diff
@@ -0,0 +1,135 @@
+module.exports = {
```

Comment:
@richm is it fine to store commitlint config in `.github` dir?

Reply:
> @richm is it fine to store commitlint config in `.github` dir?

I think `playbooks/files/` is more appropriate - especially if we implement this checking in `tox` or in some other git pre-commit check.